components/common/StarRating.tsx

```diff
@@ -1,11 +1,13 @@
 'use client';
 
+import clsx from 'clsx';
 import { useState } from 'react';
 
 type StarRatingProps = {
   rating?: number;
   isInteractive?: boolean;
   onRatingChange?: (rating: number) => void;
+  size?: string;
```

Comment:
small 이란 값을 코드를 봐야 알 수 있으니 자동완성 가능하도록 문자열 리터럴 타입 써보시는 건 어떤가요?
```suggestion
  size?: 'default' | 'small';
```